test/integration/oidc/testdata/ca.crt

```diff
@@ -0,0 +1,38 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Generate all certs using the client-go cert helpers.  Do not hard code certs in test data.

Reply:
@enj I thought that's normal in that project looking to that testdata in another OIDC tests:
https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/testdata
But agreed, such data shouldn't be hardcoded, I'll change that